packages/client/modules/userDashboard/components/OrgAuthentication/OrgAuthenticationSSOFrame.tsx

```diff
@@ -58,14 +61,42 @@ const ContactLink = styled('a')({
   }
 })
 
+const DomainWrapper = styled('div')({
+  display: 'flex',
+  flexDirection: 'row',
+  [mobileBreakpoint]: {
+    flexDirection: 'column'
+  }
+})
+
+const DomainChips = styled('span')({
+  backgroundColor: PALETTE.SLATE_200,
+  borderRadius: '12px',
+  padding: '4px 12px',
+  margin: '8px 8px 4px 0',
+  fontSize: '14px',
+  fontWeight: 600,
+  color: PALETTE.SLATE_800,
+  width: 'fit-content'
+})
+
 interface Props {
+  samlInfo:
```

Comment:
+1 could we use a relay fragment here instead of manually typing out what we expect?